### DISCUSSION TITLE
Copy filter as JSON from App filter component (to re-use in code)

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#18913] [Feature Requests]
### Summary

A lot of times we first test a filter in the admin app via the filter-component and then re-create this filter in our code (either manually or by checking the request in the inspector).
It would be a great addition if we could just copy the Filter-JSON directly from the filter component to re-use it in our code

![Bildschirmfoto 2023-06-15 um 15 31 15](https://github.com/directus/directus/assets/46492597/176a74eb-887a-4984-9ca7-275028abebca)

### Motivation

Enhance the developer experience by building filters visually from the app and export the code.

### Detailed Design

Create a JSON-Filter that can be copied directly from the filter

### Must Have:
- Generated Filter
- Copy button

### Drawbacks

/

### Alternatives

/

### Adoption Strategy

/

### Unresolved Questions

Possible differences between Rest-API and GraphQL

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
